src/xcode/ENA/ENA/Source/Services/DCCValidationService/Model/DCCValidationProgress.swift

```diff
@@ -0,0 +1,15 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+
+struct DCCValidationProgress {
```

Comment:
Isn't this more of a `DCCValidationResult`? Or maybe `DCCValidationReport`, then we don't have the naming confusion with the Swift `Result` type

Reply:
We were not sure, but because we can return this thing also when it is not reached to the rule set validations, we thought more about a progress. Yeah but to avoid swift keywords, `Report` would be fine for me, too.